code/datums/mutations/cluwne.dm

```diff
@@ -0,0 +1,57 @@
+/datum/mutation/human/cluwne
+	name = "Cluwne"
+	desc = "Turns a person into a Cluwne, a poor soul cursed to a short and miserable life by the honkmother."
+	quality = NEGATIVE
+	locked = TRUE
+
+/datum/mutation/human/cluwne/on_acquiring(mob/living/carbon/owner)
+	if(..())
+		return
+	owner.dna.add_mutation(/datum/mutation/human/epilepsy)
+	owner.setOrganLoss(ORGAN_SLOT_BRAIN, 199)
+
+	playsound(owner.loc, 'sound/misc/honk_echo_distant.ogg', 50, 1)
+	owner.equip_to_slot_or_del(new /obj/item/storage/backpack/clown(owner), ITEM_SLOT_BACK) // this is purely for cosmetic purposes incase they aren't wearing anything in that slot
+	if(!istype(owner.wear_mask, /obj/item/clothing/mask/gas/cluwne))
+		if(!owner.dropItemToGround(owner.wear_mask, force = TRUE))
+			qdel(owner.wear_mask)
+		owner.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/cluwne(owner), ITEM_SLOT_MASK)
+
+	if(ishuman(owner))
+		var/mob/living/carbon/human/victim = owner
+		if(!istype(victim.w_uniform, /obj/item/clothing/under/rank/civilian/cluwne))
+			if(!victim.dropItemToGround(victim.w_uniform))
+				qdel(victim.w_uniform)
+			victim.equip_to_slot_or_del(new /obj/item/clothing/under/rank/civilian/cluwne(victim), ITEM_SLOT_ICLOTHING)
+		if(!istype(victim.shoes, /obj/item/clothing/shoes/cluwne))
+			if(!victim.dropItemToGround(victim.shoes))
+				qdel(victim.shoes)
+			victim.equip_to_slot_or_del(new /obj/item/clothing/shoes/cluwne(victim), ITEM_SLOT_FEET)
+		owner.equip_to_slot_or_del(new /obj/item/clothing/gloves/color/white(owner), ITEM_SLOT_GLOVES) // ditto
```

Comment:
early returns
also how could the owner not be human isn't that checked in our parent proc's `on_acquiring`